### PR TITLE
Bp styling/homepage

### PR DESCRIPTION
**Pull request by BritneyParkerson**
### Location:
-RenderHomePage.js

### Details:
1) Added some styling to the homepage. Attempting to add some other pages and links.
-------------------------
## Committed Features:
1 | First PR

### CODE REVIEW

**Review by slroberts — APPROVED** [review]
Nice work with the styling!

File: src/components/pages/Home/RenderHomePage.js [hunk 1]
```diff
@@ -29,6 +33,76 @@ function RenderHomePage(props) {
...
+        size="large"
+        align="center"
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          margin: '2%',
+          textAlign: 'center',
+        }}
+      >
+        <Form class="search" action="">
```
Comment (slroberts) [inline src/components/pages/Home/RenderHomePage.js]: There is a reusable SearchForm component in the common folder that you can use to replace this one here.

Comment (BritneyParkerson) [reply src/components/pages/Home/RenderHomePage.js]: Oops I thought I deleted that one from the code after seeing your code. I’ll fix it when I code later.

**Review by codyt11 — APPROVED** [review]
This is a pretty extensive PR, I would like to see fewer files and lines of code to review. I'll go ahead and approve this time but please next time break up PRs into smaller request! Great work though I love the carousel!

**Pull request merged**